catchpoint/dashboards/Catchpoint-Response_Size.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
Same comment here as the other dashboards! The sanitize dashboard script should clean things up